### PR TITLE
Invalid player upon initializing fix

### PR DESCRIPTION
**Pull request by overlordbleck**

### CODE REVIEW

File: lua/autorun/client/cl_apple.lua [hunk 1]
```diff
@@ -6,6 +6,7 @@ end)
 
 net.Receive( "cfc_playerinitialspawn", function( len )
```
Comment (sarahsturgeon) [inline lua/autorun/client/cl_apple.lua:7]: Why does this take an entity? Couldn't we just send the info about the player over to the client? 

Hell, we could even send the entire string to print to the client's chat box.

Can we do this from the serverside actually? Or can we not color the text with `ply:ChatPrint` ?

Comment (overlordbleck) [reply lua/autorun/client/cl_apple.lua:7]: It sends the entity so I don't have to send everything else like the players name, their team, stuff like that. And no, you can't color with `ply:ChatPrint` .

Comment (sarahsturgeon) [reply lua/autorun/client/cl_apple.lua:7]: Well, if we got the info on the serverside then I think we'd have a better chance of actually getting the player's info. Just seems like we shouldn't send an entire entity to the client when we only need a few bits of text

**Review by sarahsturgeon — APPROVED** [review]
nailed it

**Pull request merged**